subprojects/docs/src/docs/userguide/running-builds/init_scripts.adoc

```diff
@@ -40,9 +40,9 @@ One main limitation of init scripts is that they cannot access classes in the `b
 There are several ways to use an init script:
 
 * Specify a file on the command line. The command line option is `-I` or `--init-script` followed by the path to the script. The command line option can appear more than once, each time adding another init script. The build will fail if any of the files specified on the command line does not exist.
-* Put a file called `init.gradle` (or `init.gradle.kts` for Kotlin) in the `__USER_HOME__/.gradle/` directory.
-* Put a file that ends with `.gradle` (or `.init.gradle.kts` for Kotlin) in the `__USER_HOME__/.gradle/init.d/` directory.
-* Put a file that ends with `.gradle` (or `.init.gradle.kts` for Kotlin) in the `__GRADLE_HOME__/init.d/` directory, in the Gradle distribution. This allows you to package up a custom Gradle distribution containing some custom build logic and plugins. You can combine this with the <<gradle_wrapper.adoc#gradle_wrapper,Gradle wrapper>> as a way to make custom logic available to all builds in your enterprise.
+* Put a file called `init.gradle` (or `init.gradle.kts` for Kotlin) in the `__$USER_HOME__/.gradle/` directory.
```

Comment:
Should `GRADLE_USER_HOME` be in this list? Is location searched for init scripts?

Reply:
No, init scripts are not searched for in `GRADLE_USER_HOME`

Reply:
Yes.
The default for `GRADLE_USER_HOME` is `$HOME/.gradle/`